ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -170,44 +177,49 @@ export default function EditGasPopover({
 
   const footerButtonText = confirmButtonText || t('save');
   return (
-    <Popover
-      title={title}
-      onClose={closePopover}
-      className="edit-gas-popover__wrapper"
-      footer={
-        <Button
-          type="primary"
-          onClick={onSubmit}
-          disabled={hasGasErrors || balanceError || !txParamsHaveBeenCustomized}
-        >
-          {footerButtonText}
-        </Button>
-      }
-    >
-      <div style={{ padding: '0 20px 20px 20px', position: 'relative' }}>
-        {process.env.IN_TEST ? null : <LoadingHeartBeat />}
-        <EditGasDisplay
-          dappSuggestedGasFeeAcknowledged={dappSuggestedGasFeeAcknowledged}
-          setDappSuggestedGasFeeAcknowledged={
-            setDappSuggestedGasFeeAcknowledged
-          }
-          estimatedMinimumNative={estimatedMinimumNative}
-          gasPrice={gasPrice}
-          setGasPrice={setGasPrice}
-          gasLimit={gasLimit}
-          setGasLimit={setGasLimit}
-          properGasLimit={properGasLimit}
-          mode={mode}
-          transaction={updatedTransaction}
-          onManualChange={onManualChange}
-          minimumGasLimit={minimumGasLimitDec}
-          balanceError={balanceError}
-          txParamsHaveBeenCustomized={txParamsHaveBeenCustomized}
-          gasErrors={gasErrors}
-          {...editGasDisplayProps}
-        />
-      </div>
-    </Popover>
+    <Modal isOpen onClose={closePopover} className="edit-gas-popover__wrapper">
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader onClose={closePopover} marginBottom={4}>
+          {title}
+        </ModalHeader>
+        <div style={{ padding: '10px 20px 20px 20px', position: 'relative' }}>
+          {process.env.IN_TEST ? null : <LoadingHeartBeat />}
+          <EditGasDisplay
+            dappSuggestedGasFeeAcknowledged={dappSuggestedGasFeeAcknowledged}
+            setDappSuggestedGasFeeAcknowledged={
+              setDappSuggestedGasFeeAcknowledged
+            }
+            estimatedMinimumNative={estimatedMinimumNative}
+            gasPrice={gasPrice}
+            setGasPrice={setGasPrice}
+            gasLimit={gasLimit}
+            setGasLimit={setGasLimit}
+            properGasLimit={properGasLimit}
+            mode={mode}
+            transaction={updatedTransaction}
+            onManualChange={onManualChange}
+            minimumGasLimit={minimumGasLimitDec}
+            balanceError={balanceError}
+            txParamsHaveBeenCustomized={txParamsHaveBeenCustomized}
+            gasErrors={gasErrors}
+            {...editGasDisplayProps}
+          />
+        </div>
+        <Box marginLeft={4} marginRight={4} marginTop={2} marginBottom={2}>
+          <Button
+            block
+            variant={BUTTON_VARIANT.PRIMARY}
+            onClick={onSubmit}
+            disabled={
+              hasGasErrors || balanceError || !txParamsHaveBeenCustomized
+            }
```

Comment:
We can use `size={BUTTON_SIZE.LG}` here. Currently, the default size is MD (Medium), as shown in the pull request description video. 
After making this change, the button look like this 👇 which one is better?

![after change](https://github.com/MetaMask/metamask-extension/assets/81921291/f5bb7a36-c013-41f3-9a2f-8a80ac9fce26)

Reply:
The large button should look similar to the original button. Should I proceed with updating it to that design? If you have any suggestions, I'm open to changing it if the old medium size acceptable.